src/wazuh_modules/inventory_sync/src/inventorySyncFacade.hpp

```diff
@@ -129,16 +129,15 @@ class InventorySyncFacadeImpl final
             }
             // Check if session exists.
             std::shared_lock lock(m_agentSessionsMutex);
-            if (auto it = m_agentSessions.find(end->session()); it == m_agentSessions.end())
+            auto it = m_agentSessions.find(end->session());
+            if (it == m_agentSessions.end())
             {
                 throw InventorySyncException("Session not found");
             }
-            else
-            {
-                // Handle end.
-                std::cout << "Handling end for session: " << end->session() << std::endl;
-                it->second.handleEnd(*m_responseDispatcher);
-            }
+
+            // Handle end.
+            std::cout << "Handling end for session: " << end->session() << std::endl;
```

Comment:
Do we want this console output directives here? Shouldn't we use loggerHelper class?

Reply:
+1